package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "teamsnap-ui",
+  "version": "0.1.0",
+  "description": "a CSS component library for TeamSnap",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
This section has some options to write specific action that should take place after pushing to git, installing into a project etc.. <https://docs.npmjs.com/misc/scripts>